src/com/etendoerp/dependencymanager/process/AddDependency.java

```diff
@@ -57,133 +58,405 @@ public class AddDependency extends BaseActionHandler {
   private static final String DEPENDENCY_MANAGER_TABID = "0A5D3E90916C40C2B712785CC5B113BF";
   private static final Logger log = LogManager.getLogger();
 
+  /**
+   * Gets the dependencies of the specified package.
+   *
+   * <p>If the package is not a bundle, a dependency tree is created. If it is a bundle,
+   * the dependencies are taken from the provided JSON, specifically from "_params" and "grid".</p>
+   *
+   * @param packageVersion
+   *     The version of the package to get dependencies for.
+   * @param jsonContent
+   *     The JSON with the details to build dependencies, used when the package is a bundle.
+   * @return A list of dependencies for the package.
+   * @throws JSONException
+   *     If the JSON does not have the necessary keys when the package is a bundle.
+   */
```

Comment:
I suggest adding log.debug to the getPackageDependencies method for better traceability of getting dependencies.